clients/java-logger/library/src/test/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessorTests.java

```diff
@@ -78,14 +92,31 @@ public void processShouldPopulateJustificationMetadata() throws Exception {
                         ListValue.newBuilder()
                             .addValues(Value.newBuilder().setStringValue("test-aud")))
                     .build())
+            .putFields(
+                "justs",
+                Value.newBuilder()
+                    .setListValue(
+                        ListValue.newBuilder()
+                            .addValues(Value.newBuilder().setStructValue(wantJustificationsClaim)))
+                    .build())
             .build();
     Struct wantMetadata =
         Struct.newBuilder()
             .putFields(
                 "justification", Value.newBuilder().setStructValue(wantJustification).build())
             .build();
+    RequestMetadata wantRequestMetadata =
+        RequestMetadata.newBuilder()
+            .setRequestAttributes(
+                Request.newBuilder()
+                    .setReason("[{\"category\":\"explanation\",\"value\":\"need-access\"}]")
+                    .build())
```

Comment:
We're missing negative tests - validate that the request metadata isn't populated on null/empty justs

Reply:
We have a ut setLogJustificationShouldPopulateJustificationMetadata in which request metadata is skipped when justs is null, I will create seperate ut when list is null or empty, thanks for pointing it out!